### PR TITLE
[NOMAD] Update of the C interface

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: N/NOMAD/build_tarballs.jl [hunk 1]
```diff
@@ -5,24 +5,18 @@ version = v"4.0.0"
...
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://gist.github.com/amontoison/06dac8b63424854f754264597af6b09e/raw/942aba554a82800d0c7438288d3da1a827ef2974/NOMAD.zip", "a6653af375be8006e742239af3914ba48034015d57a2be7a07fc14c6ff245d1b"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/amontoison/nomad.git","4c5511e873fc24b82f5ea9ffccd9c9912b4eebfe"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd "${WORKSPACE}/srcdir/NOMAD"
```
Comment (giordano) [inline N/NOMAD/build_tarballs.jl:13]: I assume this is now called
```suggestion
cd "${WORKSPACE}/srcdir/nomad"
```

Comment (amontoison) [reply N/NOMAD/build_tarballs.jl]: Thanks !

**Comment by giordano** [comment]
Am I correct to say that it's trying to run an executable built for the target platform?  Unfortunately we can't do that

**Comment by amontoison** [comment]
@giordano Do you have an idea why the patch cannot be applied ?

**Comment by giordano** [comment]
I can't apply it either locally, so wrong patch? :shrug: 
```
nomad% patch -p1 < compilation.patch 
patching file doc/doxygen/src/Attribute/cacheAttributesDefinition.hpp
patching file doc/doxygen/src/Attribute/displayAttributesDefinition.hpp
patching file doc/doxygen/src/Attribute/evalAttributesDefinition.hpp
patching file doc/doxygen/src/Attribute/evaluatorControlAttributesDefinition.hpp
patching file doc/doxygen/src/Attribute/pbAttributesDefinition.hpp
patching file doc/doxygen/src/Attribute/runAttributesDefinition.hpp
patching file doc/doxygen/src/CMakeLists.txt
patching file src/Attribute/cacheAttributesDefinition.hpp
patching file src/Attribute/displayAttributesDefinition.hpp
patching file src/Attribute/evalAttributesDefinition.hpp
patching file src/Attribute/evaluatorControlAttributesDefinition.hpp
patching file src/Attribute/pbAttributesDefinition.hpp
patching file src/Attribute/runAttributesDefinition.hpp
patching file src/CMakeLists.txt
Reversed (or previously applied) patch detected!  Assume -R? [n] 
```

**Comment by amontoison** [comment]
You were right Mosè, I regenerated the patch and it works.

**Pull request merged**